### PR TITLE
Portfolio profile

### PR DESCRIPTION
**Pull request by edubew**
## In this milestone  I was able to:
- Create a new branch portfolio-profile
- Create an index.html file with a navigation bar and header area for the project
- Create a style.css file and added styling to the header and navigation bar 
- Check for linters errors
- Fix linters errors in both index.html and style.css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Wiinnie's Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <a href="">John Doe</a>
+        </div>
+        <div class="menu-bar">
+            <img src="IMG1/icon.png" alt="Bars">
```
Comment (micheaol) [inline index.html:19]: Please ensure that the hamburger button has a `nav` tag.

`<header>`
         `nav`
            `Bars`
        `nav`
`</header>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+    </nav>
+    <header>
```
Comment (micheaol) [inline index.html:22]: Please ensure that the container of the section is a `section` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                <h1>Winfred Edube</h1>
+                <h2>Software Developer</h2>
+            </div>
+            <div class="image">
+               
+            </div>
+        <div class="intro-container">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+        </div>
+    </header>
```
Comment (micheaol) [inline index.html:33]: Please ensure that you stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Styling: Active, Hover and Pressed for ALL buttons.

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @edubew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — APPROVED** [review]
### Hi  @edubew 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions
- [ ] I would suggest you prevent the horizontal scroll on the page as this not a good user experience
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by edubew** [comment]
Hello @AnselemOdims 
How do I prevent a horizontal scroll on the page?

**Pull request merged**